Syntax Highlighter based on Ace editor

MilanLund opened the pull request:
### Motivation

Existing issue: https://github.com/Kentico/custom-element-samples/issues/8

### How to test

See readme on how to use it.
Test for obtaining the value: https://codepen.io/milanlund/pen/rRZRVe

Simply007 commented:
And just one note - the editor is Under the BSD license, I would add this note to the Readme.

MilanLund commented:
Hey @Simply007 ,

Many thanks for your review. Regarding the comments:

- When published the editor has already been set to read-only mode. That means you can not update its value while it is possible to interact with the editor in other ways. Hope that is okay with you. Regarding the dropdown, I updated that to become disabled when published.
- I set textmate theme and plain_text mode as defaults. 
- I updated the Readme file with the licencing note.

Please, let me know whether you are fine with the updates. Cheers!

Simply007 commented on SyntaxHighlighter/README.md:
```diff
@@ -0,0 +1,47 @@
+* Open / create a content model to which you want to add the Syntax Highlighter
+* Add **Custom element** content element
+* Open configuration of the content element
+* Use absolute URL of the index.html file hosted on your end as Hosted code URL (HTTPS)
```
Would it be possible to add the URL on this repo?

```suggestion
* Use absolute URL of the index.html file hosted on your end as Hosted code URL (HTTPS)
  * For testing purposes it is possible to use https://kentico.github.io/custom-element-samples/SyntaxHighlighter/index.html
```
